fetch_data.py

```diff
@@ -283,3 +310,134 @@ def get_from_dict(data_dict, map_list):
     Thanks https://stackoverflow.com/a/14692747/11417211
     """
     return reduce(operator.getitem, map_list, data_dict)
+
+
+def parse_meta_xml_str(meta_xml_str):
+    """Parse the DescribeCoverage request to get the XML and
+    restructure to dict
+    
+    Args:
+        meta_xml_str (str): decoded text XML response from
+            DescribeCoverage WCS query
+    """
+    meta_xml = ET.ElementTree(ET.fromstring(meta_xml_str))
+    # wow xml
```

Comment:
D:

Reply:
He I think the only option for DescribeCoverage requests is XML!! sad